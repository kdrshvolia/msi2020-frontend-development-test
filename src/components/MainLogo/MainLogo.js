import React from 'react';import PropTypes from 'prop-types';const MainLogo = ({ text }) => {  return <div>{text}</div>;};MainLogo.propTypes = {  text: PropTypes.string.isRequired,};export default MainLogo;