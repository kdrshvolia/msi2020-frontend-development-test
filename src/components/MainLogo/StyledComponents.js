import styled from 'styled-components';

const StyledMainLogo = styled.h3`
  font-weight: bold;
  font-size: 20px;
  line-height: 28px;
  margin: 0;
`;

export { StyledMainLogo };
