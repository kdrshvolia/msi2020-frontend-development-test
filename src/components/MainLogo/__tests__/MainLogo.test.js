import React from 'react';import renderer from 'react-test-renderer';import MainLogo from '../MainLogo';describe('MainLogo', () => {  it('renders correctly', () => {    const rendered = renderer.create(      <MainLogo text="some text" />,    );    expect(rendered.toJSON()).toMatchSnapshot();  });});