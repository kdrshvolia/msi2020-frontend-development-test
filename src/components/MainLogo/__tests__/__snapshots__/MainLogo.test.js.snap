// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLogo renders correctly 1`] = `
<h3
  className="sc-AxjAm llhpTt"
>
  some text
</h3>
`;
